README.md

```diff
@@ -1 +1,104 @@
-# capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone
+
+> online website for developersweek confrence
+
+Built from this [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+## Live Demo
+
+[Live Demo link](https://shahnawaza75.github.io/capstone/)
+
+## Video
+
+In the following [link](https://www.loom.com/share/345b67d04e1a4d748a80a9739e4fa760) I am explaining the capstone project.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- ESLint
+
+### Prerequisites
+
+The basic requirements for building the executable are:
+
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor
+- Node Package Manager (For installing packages like Lighthous, webhint & stylelint used for checking for debugging bad codes before deployment)
+
+#### Cloning the project
+
+```
+git clone: https://github.com/shahnawaza75/capstone.git
+```
+
+## Getting packages and debuging with Webhint
+
+```
+npm init -y
+npm install --save-dev hint@6.x
+```
+
+##### For validation detection using Webhint Run
+
+```
+npx hint .
+```
+
+##### from parent source directory
+
+## Getting packages and debuging with Stylelint
+
+```
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+##### For validation detection using Stylelint Run
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+##### from parent source directory
+
+## Getting packages and debuging with Stylelint
+
+```
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+```
+
+##### For validation detection using Stylelint Run
+
+```
+npx eslint .
+```
+
+##### from parent source directory
+
+## Authors
+
+👤 **Shahnwaz Ali**
+
+- GitHub: [@Shahnawaza75](https://github.com/shahnawaza75/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip Cindy Shin the author of the original design
```

Comment:
Great 🤓 Addressed 

### OPTIONAL 

You could provide the link to Cindy's design 🧐